osmchadjango/changeset/templates/changeset/changeset_list.html

```diff
@@ -340,6 +309,24 @@
   }
 
 });
-</script>
+document.getElementById('date__gte').addEventListener('input', function () {
+   var val = document.getElementById('date__gte').value;
+   var today = moment().format('MM-DD-YYYY');
+   var tomorrow = moment().add(1, 'day').format('MM-DD-YYYY');
+
+   if (val === 'Today') {
+    document.getElementById('date__gte').value = today;
+    document.getElementById('date__lte').value = tomorrow;
+   } else if (val === 'Last 7 days') {
+    var sevenDaysAgo = moment().subtract(6, 'day').format('MM-DD-YYYY');
+    document.getElementById('date__gte').value = sevenDaysAgo;
+    document.getElementById('date__lte').value = tomorrow;
+   } else if (val === 'Last 30 days') {
+    var thirtyDaysAgo = moment().subtract(29, 'days').format('MM-DD-YYYY');
+    document.getElementById('date__gte').value = thirtyDaysAgo;
+    document.getElementById('date__lte').value = tomorrow;
+   }
+});
```

Comment:
Would be nice to stick to using a jQuery style everywhere for consistency -- so like:

`$('#date__gte').val(thirtyDaysAgo);` instead of `document.getElementById.....`